docs/getting_started.md

```diff
@@ -22,12 +22,7 @@ This will create a new namespace, `argocd`, where Argo CD services and applicati
     The installation manifests include `ClusterRoleBinding` resources that reference `argocd` namespace. If you are installing Argo CD into a different
     namespace then make sure to update the namespace reference.
 
-If you are not interested in UI, SSO, multi-cluster features then you can install [core](operator-manual/installation.md#core) Argo CD components only:
-
-```bash
-kubectl create namespace argocd
-kubectl apply -n argocd -f https://raw.githubusercontent.com/argoproj/argo-cd/stable/manifests/core-install.yaml
-```
+If you are not interested in UI, SSO, multi-cluster features then you can install [core](operator-manual/core/#installing) Argo CD components only.
```

Comment:
After a second thought, do you think it would be useful to turn this block into a tip to highlight the core installation?  
e.g. 
```
!!! tip
    If you are not interested in UI, SSO, multi-cluster features then you can install [core](operator-manual/core/#installing) Argo CD components only.
```

Reply:
Yeah, I like that idea!
```suggestion
!!! tip
    If you are not interested in UI, SSO, and multi-cluster features, then you can install only the [core](operator-manual/core/#installing) Argo CD components.
```